dapp/src/components/TransactionModal/ActiveStakingStep/StakingErrorModal/RetryModal.tsx

```diff
@@ -29,7 +31,13 @@ const getCounterData = (minutes: string, seconds: string) => {
   return { label, progressPercent }
 }
 
-export default function RetryModal({ retry }: { retry: () => void }) {
+export default function RetryModal({
+  isLoading,
+  retry,
+}: {
+  isLoading: boolean
```

Comment:
NB:
Let's make the type definition explicit for consistency. 
Like `type RetryModalProps = { ... }`

Reply:
https://github.com/thesis/acre/pull/828/commits/ebc043b071a89ca671b95455d652755b850f1cda